source/server/http/admin.cc

```diff
@@ -956,6 +956,35 @@ uint64_t PrometheusStatsFormatter::statsAsPrometheus(
     response.add(fmt::format("{0}_count{{{1}}} {2}\n", metric_name, tags, stats.sampleCount()));
   }
 
+  for (const auto& histogram : histograms) {
```

Comment:
I am confused with this. This change already exists https://github.com/envoyproxy/envoy/blob/master/source/server/http/admin.cc#L1003? and git diff does not match?

Reply:
hey, was computing the quantiles from the histogram itself, that's why duplicated the block while testing. also closing the PR(plz check below)

thanks for your time